docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[release-notes-header-8.12.0]]
+== 8.12
+
+[discrete]
+[[release-notes-8.12.0]]
+=== 8.12.0
+
+[discrete]
+[[known-issue-8.12.0]]
+==== Known issues
+
+
+// tag::known-issue-173958[]
+[discrete]
+.[Title]
+[%collapsible]
+====
+*Details* +
+When editing the Alerts page KQL query bar filter or editing the KQL query bar filter on the rule edit page, you might encounter a UI bug requiring you to select a data view to proceed. 
```

Comment:
The **Workaround** heading is rendering on the same line this paragraph, but this adds a soft return at the end:
```suggestion
When editing the Alerts page KQL query bar filter or editing the KQL query bar filter on the rule edit page, you might encounter a UI bug requiring you to select a data view to proceed. +
```